frontend/www/js/omegaup/arena/qualitynomination_popup.js

```diff
@@ -14,14 +14,19 @@ OmegaUp.on('ready', function() {
       return createElement('qualitynomination-popup', {
         props: {
           nominated: this.nominated,
+          nominatedBeforeAC: this.nominatedBeforeAC,
           solved: this.solved,
+          tried: this.tried,
           dismissed: this.dismissed,
+          dismissedBeforeAC: this.dismissedBeforeAC,
           canNominateProblem: this.canNominateProblem,
         },
         on: {
           submit: function(ev) {
             let contents = {};
-
+            if (!ev.solved && ev.tried) {
+              contents['before_ac'] = true;
```

Comment:
```suggestion
              contents.before_ac = true;
```

en JavaScript se prefiere usar la notación de punto sobre la de arreglo para objetos. así eres consistente con el resto de los lugares donde se modifica `contents`.

Reply:
De acuerdo!